src/pages/gallery/index.mdx

```diff
@@ -14,549 +23,513 @@ visible should contribute to the correct total statement.” — Eliot Noyes
 
-![Hundred Percent Club](images/HPC_Tote_.jpg)
+![Hundred Percent Club](../images/HPC_Tote_.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Db2" title="IBM Db2" col={4}>
 
-![IBM Db2](images/IBM_Db2_Social_Tile.jpg)
+![IBM Db2](../images/IBM_Db2_Social_Tile.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Brochures.jpg)
+![IBM Services](../images/IBM_Services_Brochures.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Tote.jpg)
+![IBM Services](../images/IBM_Services_Tote.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Watch.jpg)
+![IBM Services](../images/IBM_Services_Watch.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={8}>
 
-![IBM Services](images/IBM_Services_Keynote.jpg)
+![IBM Services](../images/IBM_Services_Keynote.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={8}>
 
-![IBM Services](images/IBM_Services_Progress_Animation.gif)
+![IBM Services](../images/IBM_Services_Progress_Animation.gif)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Digital.jpg)
+![IBM Services](../images/IBM_Services_Digital.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Photography.jpg)
+![IBM Services](../images/IBM_Services_Photography.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Presentation.jpg)
+![IBM Services](../images/IBM_Services_Presentation.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Brand.jpg)
+![IBM Services](../images/IBM_Services_Brand.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Services" title="IBM Services" col={4}>
 
-![IBM Services](images/IBM_Services_Newsletter.jpg)
+![IBM Services](../images/IBM_Services_Newsletter.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Research" title="IBM Research" col={8}>
 
-![IBM Research](images/IBM_Research_Almaden.jpg)
+![IBM Research](../images/IBM_Research_Almaden.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Digital Analytics" title="IBM Digital Analytics" col={8}>
 
-![IBM](images/IBM_Digital_Analytics_Prototype.jpg)
+![IBM](../images/IBM_Digital_Analytics_Prototype.jpg)
 
 </ImageGalleryImage>
 <ImageGalleryImage alt="IBM Security" title="IBM Security" col={4}>
 
-<Video
-  playsInline
-  autoPlay
-  loop
-  poster="images/IBM_Security_Cloud.jpg)"
-  src="videos/IBM_Security_Cloud.mp4"
-/>
+<GalleryVideo src={ibmSecurityCloud} />
```

Comment:
same question about the `poster`

Reply:
I think the poster was removed since these are autoplay? @vpicone

Reply:
@alisonjoseph yeah, we want to start the video immediately so there’s no advantage to fetching the poster. Could definitely try it on a throttled connection to see if it makes a dif tho.